### PR TITLE
UI tests: add assert for uploaded disk status

### PR DESCRIPTION
**Pull request by ljelinkova**

### CODE REVIEW

File: ost_utils/selenium/page_objects/DisksListView.py [hunk 1]
```diff
@@ -37,6 +37,23 @@ def is_copy_button_enabled(self):
...
 
+    def get_status(self, entity_name):
+        names_to_ids = self.ovirt_driver.retry_if_stale(self._get_entity_names_to_ids)
+
+        if entity_name in names_to_ids:
+            # find the id of the name column (column 0) and replace 1 by 10 to
+            # get the id of the status column (column 10)
+            name_id = names_to_ids[entity_name]
+            name_id_col_index = name_id.find("0_row")
+            status_id = name_id[:name_id_col_index] + '1' + name_id[name_id_col_index:]
```
Comment (tinez) [inline ost_utils/selenium/page_objects/DisksListView.py:48]: Hmmm wouldn't `status_id = name_id.replace("0_row", "10_row")` work?

**Review by tinez — COMMENTED** [review]
Some nits, but generally LGTM

File: basic-suite-master/test-scenarios/test_100_basic_ui_sanity.py [hunk 2]
```diff
@@ -479,6 +479,7 @@ def test_disks(ovirt_driver, selenium_browser_name, image_local_path):
 
     image_name = "{}-{}".format(selenium_browser_name, int(time.time()))
     disks_list_view.upload(image_local_path, image_name)
+    assert assert_utils.equals_within_long(lambda: disks_list_view.get_status(image_name), 'OK')
```
[the remark this answers is not in the document]
Comment (ljelinkova) [reply basic-suite-master/test-scenarios/test_100_basic_ui_sanity.py:482]: Is 3 mins to upload disk enough?

File: ost_utils/selenium/page_objects/DisksListView.py (hunk 1 above)
Comment (ljelinkova) [reply ost_utils/selenium/page_objects/DisksListView.py:48]: Yes, it works :-)

File: ost_utils/selenium/page_objects/DisksListView.py [hunk 3]
```diff
@@ -37,6 +37,23 @@ def is_copy_button_enabled(self):
...
+            status_text = self.ovirt_driver.retry_if_stale(self._get_status_text, status_id)
+            return status_text
+        else:
+            raise Exception('No ' + self.entity_type + ' with the name ' + entity_name + ' found')
+
+    def _get_status_text(self, entity_status_id):
```
[the remark this answers is not in the document]
Comment (ljelinkova) [reply ost_utils/selenium/page_objects/DisksListView.py:54]: Ok.

File: basic-suite-master/test-scenarios/test_100_basic_ui_sanity.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (tinez) [reply basic-suite-master/test-scenarios/test_100_basic_ui_sanity.py]: I think so - looking at the current execution time it's around 20-30s

[the remark this answers is not in the document]
Comment (ljelinkova) [reply basic-suite-master/test-scenarios/test_100_basic_ui_sanity.py]: OK

File: ost_utils/selenium/page_objects/DisksListView.py [hunk 4]
```diff
@@ -37,6 +37,21 @@ def is_copy_button_enabled(self):
     def is_upload_button_enabled(self):
         return self.ovirt_driver.is_button_enabled('Upload')
 
+    def get_status(self, entity_name):
+        names_to_ids = self.ovirt_driver.retry_if_stale(self._get_entity_names_to_ids)
```
[the remark this answers is not in the document]
Comment (ljelinkova) [reply ost_utils/selenium/page_objects/DisksListView.py:41]: OK, do I understand you correctly that you are OK with changing the method from private to public in different patch?

**Review by tinez — APPROVED** [review]

**Pull request merged**